From 698a95edb64941495b781451a1e323155fe87c5f Mon Sep 17 00:00:00 2001
From: Nuno Araujo <nuno.araujo@russo79.com>
Date: Wed, 16 Jan 2013 09:27:03 +0100
Subject: [PATCH] Fix the build with automake 1.13

In Automake 1.13, the long-deprecated macro AM_CONFIG_HEADER (deprecated
since 2002) has been removed in favour of AC_CONFIG_HEADERS.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index fad886d..83176e4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2,7 +2,7 @@
 
 AC_PREREQ(2.59)
 AC_INIT([xrdp], [0.6.0], [xrdp-devel@lists.sourceforge.net])
-AM_CONFIG_HEADER(config_ac.h:config_ac-h.in)
+AC_CONFIG_HEADERS(config_ac.h:config_ac-h.in)
 AM_INIT_AUTOMAKE([1.6 foreign])
 AC_PROG_CC
 AC_C_CONST
-- 
1.8.1.1

